elixir/apps/domain/lib/domain/ops.ex

```diff
@@ -25,4 +25,21 @@ defmodule Domain.Ops do
         })
     end)
   end
+
+  def provision_support_by_account_slug(account_slug) do
+    Domain.Repo.transaction(fn ->
+      {:ok, account} = Domain.Accounts.fetch_account_by_id_or_slug(account_slug)
+      {:ok, providers} = Domain.Auth.list_active_providers_for_account(account)
+      magic_link_provider = Enum.find(providers, fn provider -> provider.adapter == :email end)
```

Comment:
@jamilbk keep in mind that this provider can be disabled by the admin, but I think this will suffice for now.

What we actually want is read-only account admin impersonation and an admin panel.